### PR TITLE
Make an internal function static.

### PR DESCRIPTION
**Pull request by songy23**
A follow-up of https://github.com/census-instrumentation/opencensus-java/pull/1372.

### CODE REVIEW

File: impl_core/src/main/java/io/opencensus/implcore/stats/MetricUtils.java [hunk 1]
```diff
@@ -74,15 +74,7 @@ static Type getType(Measure measure, Aggregation aggregation) {
...
-        new Function<Aggregation, Type>() {
-          @Override
-          public Type apply(Aggregation arg) {
-            if (arg instanceof Aggregation.Mean) {
-              return Type.CUMULATIVE_DOUBLE; // Mean
-            }
-            throw new AssertionError();
-          }
-        });
+        AGGREGATION_TYPE_DEFAULT_FUNCTION);
```
Comment (bogdandrutu) [inline impl_core/src/main/java/io/opencensus/implcore/stats/MetricUtils.java]: You should make all the functions static if you want to have effect (e.g. Functions.returnConstant(Type.CUMULATIVE_DOUBLE) or Functions.returnConstant(Type.CUMULATIVE_INT64))

Comment (songy23) [reply impl_core/src/main/java/io/opencensus/implcore/stats/MetricUtils.java]: I see, updated.

File: impl_core/src/main/java/io/opencensus/implcore/stats/MetricUtils.java [hunk 2]
```diff
@@ -93,5 +85,34 @@ public Type apply(Aggregation arg) {
...
+      Functions.returnConstant(Type.CUMULATIVE_DISTRIBUTION);
+
+  private static final Function<Object, Type> TYPE_GAUGE_DOUBLE_FUNCTION =
+      Functions.returnConstant(Type.GAUGE_DOUBLE);
+
+  private static final Function<Object, Type> TYPE_GAUGE_INT64_FUNCTION =
+      Functions.returnConstant(Type.GAUGE_INT64);
+
+  private static final Function<Object, Type> TYPE_UNRECOGNIZED_FUNCTION =
+      Functions.<Type>throwAssertionError();
```
Comment (bogdandrutu) [inline impl_core/src/main/java/io/opencensus/implcore/stats/MetricUtils.java]: throwAssertionError already returns the same instance all the time, don't need to make it static.

Comment (songy23) [reply impl_core/src/main/java/io/opencensus/implcore/stats/MetricUtils.java]: I see - this doesn't make a difference so I'll leave it as-is :)

**Pull request merged**